CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ## Unreleased
 
+To collect JS dependencies on iOS add `../node_modules/@sentry/react-native/scripts/collect-modules.sh` at the end of
+the `Bundle React Native code and images` build phase. The collection only works on Release builds.
```

Comment:
This should be a subitem of:

> JS Runtime dependencies are sent in Events

Also, I'd give more instructions, only the name of the section in the build phase isn't enough, file name, expected result etc would be better, or write in the docs and link the docs here, example: https://docs.sentry.io/platforms/react-native/troubleshooting/#react-native-069-and-higher

Reply:
I will for sure write a docs fore it and link it. I just thought I'll write it here short and detailed in docs.

Reply:
Writing here makes sense, but the position should be under the feature anyway.